README.md

```diff
@@ -64,10 +66,13 @@ upon detection.
 
 For instance, if a new Release were to be created in the
 [Specta](https://github.com/0xSplits/specta) repository, then Kayron will
-generate all CloudFormation templates and reconcile them against AWS, which will
-then result in a stack update, because the Specta version changed in the
-generated templates. Kayron may also keep the deployment status up to date in
-Github.
+generate all CloudFormation templates anew using the changed resource details in
+the underlying cache channels, and reconcile the resulting CloudFormation
+templates against AWS, if the specified policies permit the proposed update.
+This process will then eventually result in a stack update in CloudFormation, so
+that Kayron may also keep the respective deployment status up to date in Github.
+
+![Operator Design](.github/assets/Operator-Design.svg)
```

Comment:
While I was figuring out the internal design for the operator's reconciliation, I drew up this diagram to guide the planned implementation.